server/service/integration_mdm_test.go

```diff
@@ -159,10 +162,20 @@ func (s *integrationMDMTestSuite) SetupSuite() {
 						ctx, name, s.T().Name(), 1*time.Hour, ds, ds,
 						schedule.WithLogger(logger),
 						schedule.WithJob("manage_profiles", func(ctx context.Context) error {
+							if s.onProfileScheduleDone != nil {
+								s.onProfileScheduleDone()
```

Comment:
Maybe we could rename this to be `onProfileJobDone` and potentially have it take an `id` parameter to denote which job is being signaled done?

Reply:
what are you envisioning for that?

Only thing that comes to mind is that would be more efficient to only wait for the job you care about, but not sure if the milliseconds are worth the extra complexity.

Reply:
We don't need the `id` bit now. Maybe just renaming it would help future folks think about how to account there being multiple jobs on the schedule now.

Reply:
okay, that sounds good! Pushed a rename